recipe/meta.yaml

```
@@ -0,0 +1,127 @@
+{% set version = "2.3.1" %}
```

Looks like 2.3.2 is out now. ZLIB ABI [unchanged](https://github.com/zlib-ng/zlib-ng/blob/6d9f3dc072369dc719a5fbe71d4e086a96a680bd/zlib.h.in#L61).
```suggestion
{% set version = "2.3.2" %}
```

---

Yeah, I'll rebase since it doesn't apply anymore.